packages/babel-parser/src/parser/expression.js

```diff
@@ -2389,12 +2391,22 @@ export default class ExpressionParser extends LValParser {
     checkKeywords: boolean,
     isBinding: boolean,
   ): void {
-    if (this.prodParam.hasYield && word === "yield") {
-      this.raise(startLoc, Errors.YieldBindingIdentifier);
+    // All JavaScript reserved words are not longer than 10 characters.
```

Comment:
Nit:

```suggestion
    // Every JavaScript reserved word is or shorter than 10 characters.
```

Reply:
I think forgot a word there. Maybe `Every JavaScript reserved word is 10 characters or shorter.`?

Reply:
```suggestion
    // Every JavaScript reserved word is 10 characters or less.
```